extensions_admin/pulp_rpm/extensions/admin/upload/package.py

```diff
@@ -154,6 +175,7 @@ def _generate_unit_key(rpm_filename):
 
     # Expected unit key fields:
     # "name", "epoch", "version", "release", "arch", "checksumtype", "checksum"
+    # Includes one more extra for DRPM specification "isDRPM"
```

Comment:
That's leftover after type checking on L#89 and L#109 I than realised there is a better way and removed it.